src/routes/CommunitySettings/CommunitySettingsTab/CommunitySettingsTab.scss

```diff
@@ -95,7 +95,8 @@
 
 label {
   color: $color-regent-70;
-  margin-left: 10px;
+  // LEJ: Commenting-out as has global impact
```

Comment:
why not just change 'label' to a classname?

Reply:
Yes, certainly. That does require making updating to this feature area and doing a more comprehensive review of the impact of removing the global style throughout, but yes I can and will do that.

Reply:
hmm yeah, fair enough, would rather get this out right now

Reply:
NOTE: If this is going out as is, previous to the profile design update (in progress), Skill & Interests won't currently be appearing anywhere on the Profile.

I'm ok with that, the design update won't be far behind, but worth pointing-out.

---
... and yeah, :) basically that was the logic I was following as well. I did make better a few things in the course of this branch as it was but needed to stop there.